app/screens/receive-bitcoin-screen/ReceiveBitcoinScreenTypes.d.ts

```diff
@@ -0,0 +1,5 @@
+interface IGetFullUri {
```

Comment:
Maybe this should just be a type  instead of an interface since we aren't extending it? `GetFullUriInput`

Reply:
Sure, I can update it to type.
Would that be the rule of thumb for using types or interfaces?

Reply:
It's a complicated topic [it seems](https://stackoverflow.com/questions/37233735/typescript-interfaces-vs-types), and we shouldn't take my word as gospel. @samerbuna

Reply:
It's possible I was wrong because further down in [the link](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces) it says: `For the most part, you can choose based on personal preference, and TypeScript will tell you if it needs something to be the other kind of declaration. If you would like a heuristic, use interface until you need to use features from type.`

Reply:
Indeed, for the most part it's personal preference, so it's fine either way. I'll just follow the project setup 👌

My force of habit is to use `interface` when it's literally an interface to talk to some code.